tests/slsa_analyzer/checks/test_provenance_available_check.py

```diff
@@ -35,17 +35,14 @@ def __init__(self, profile: dict):
         super().__init__(profile)
         self.release = {
             "assets": [
-                {"name": "attestation.intoto.jsonl", "url": "URL", "size": "10"},
-                {"name": "artifact.txt", "url": "URL", "size": "10"},
+                {"name": "attestation.intoto.jsonl", "url": "URL", "size": 10},
```

Comment:
The `"size"` field here is supposed to be a number, not a string. I have checked the [GitHub docs](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release) and tried requesting the API myself.